doc/configuring.md

```diff
@@ -0,0 +1,361 @@
+
+We suggest naming at the first sequence in each FASTA file with the identifier at the start, space delimited - for example `MY_ID_1.1 first_sequence_name is very special` .
+
+### Creating one or more sourmash databases
+
+You'll need to provide at least one sourmash database for your private collection to genome-grist under the config parameter `private_databases`, which takes a list of paths to sourmash database locations.
+
+Sketch all your genomes with the following command:
+```
+sourmash sketch dna -p k=31,scaled=1000
+```
+
+If you've named your genomes so that the first sequence contains the identifier, you can add `--name-from-first` and then the sequences will be named the right thing for the next step.
+
+If not, you'll need to manually adjust the names of the signatures produced by `sourmash sketch`. (You can do this with `sourmash sig rename`, but there's no simple way to do this in bulk.)
+
+Once you have all your genome signatures, you can combine them into a single file with 
+
+```
+sourmash index output.sbt.zip *.sig
+```
+and that will then be your sourmash database that you can cherish and treasure!
+
+If you have lots of genomes (1000 or more?) there are other approaches that might make your life more convenient; just ask for suggestions on [the sourmash issue tracker](https://github.com/dib-lab/sourmash/issues).
+
+We chose k=31 above (in the `sourmash sketch` command) because that matches our default parameters, and we have provided Genbank databases for k=31 (as well as k=21 and k=51). But the only real requirement here is that all your databases support the requested k-mer and scaled sizes.
+
+### Providing your genomes to genome-grist
+
+You'll also need to provide your genome files to genome-grist, along with their "display name", which will be used in reporting.  The information will be provided via the config parameter `private_databases_info`, which takes a list of paths to info file CSVs
+
+**First**, genome-grist needs the genomes in their own individual files, in one or more directories. The files need to be named by their identifiers in the format `{ident}_genomic.fna.gz`, and must come with an "info file" that contains their identifier, a display name, and the location of the genome file (which _must_ be named as above).
+
+genome-grist has a utility to help set this all up! The script `genome_grist.copy_private_genomes` will take in a list of FASTA files containing genome(s), read the header of the first sequence to find the identifier for that genome, and then copy it into a directory for you. (see "Step 3", below, for execution instructions for this script). It will also output a provisional info file, which you can edit.
+
+**Second**, for each genome, genome-grist also needs a separate `{ident}.info.csv` file, containing just the identifier and the display name. This needs to be in the same directory as the genome itself.
```

Comment:
again, a head could be useful. Something like "the output of the utility script `genome_grist.make_info_file` should look something like:

Reply:
added in 07786d45787